### ISSUE TITLE
LaTeX in @Chapter @Section and @Subsection in Doc.autodoc

### ISSUE THREAD

**Issue by heiderich** [gap-packages/AutoDoc#127]
I am not sure whether support for LaTeX commands in chapter, section and subsection titles is already supported by AutoDoc.

If so, how?

If not, consider this a feature request to support this.

**Comment by sebasguts** [comment]
Simple Latex commands, as in such that are rendered in html and in latex section headers could be supported using the `$` syntax. As a workaround, you can currently use the corresponding xml tag `<Math>`.

**Comment by heiderich** [comment]
The line

`@Subsection The cyclic group <Math>C_2</Math> of order two`

produced "C2" (the 2 not being a subscript) in the resulting PDF. Am I doing something wrong?

**Comment by sebasguts** [comment]
No, this is right. I will check what the limitations in GAPDoc are. We
probably also have to allow it there.

**Comment by fingolfin** [comment]
@sebasguts @heiderich What's the status of this issue?

**Comment by sebasguts** [comment]
There are now the `@Chapter`-, `@Section`-, and `@SubsectionTitle` commands that should support that syntax.

**Comment by fingolfin** [comment]
Well, what does "should support" mean? Do they or don't they? Somebody should tests this and report the result. Then, if they do work, let's add tests which verify that, then close this issue. if not, details about the resulting problems could be added.
